guides/release/getting-started/core-concepts.md

```diff
@@ -16,49 +17,103 @@ No matter how the URL gets set, the first thing that happens is that the Ember r
 
 The route handler then typically does two things:
 
-* It renders a template.
-* It loads a model that is then available to the template.
+* It loads a model.
+* It renders a template, which has access to the model.
+
+## Models
+
+Models represent persistent state.
+
+For example, a property rentals application would want to save the details of
+a rental when a user publishes it, and so a rental would have a model defining
+its details, perhaps called the _rental_ model. You may also need a _user_
+model to keep track of who is currently logged in.
+
+A model typically persists information to a web server, although models can be
+configured to save to anywhere else, such as the browser's Local Storage.
 
 ## Templates
 
-Ember uses templates to organize the layout of HTML in an application.
+Ember uses templates to build up the user interface in an application.
 
-Most templates in an Ember codebase are instantly familiar, and look like any
-fragment of HTML. For example:
+If you have writen HTML before, you already know how to write a basic Ember
+template. For example:
 
-```handlebars
+```handlebars {data-filename="app/templates/welcome.hbs"}
 <div>Hi, this is a valid Ember template!</div>
 ```
 
-Ember templates use the syntax of [Handlebars](http://handlebarsjs.com)
-templates. Anything that is valid Handlebars syntax is valid Ember syntax.
+In addition to static HTML content, Ember uses the syntax of [Handlebars](http://handlebarsjs.com)
+to describe dynamic user interface elements.
+
+For example, as mentioned before, the route handler makes the model available
+to its template:
+
+```handlebars {data-filename="app/templates/welcome.hbs"}
+{{!-- The model for this route is the current user --}}
 
-Templates can also display properties provided to them from their context, which is either a component or a route's controller. For example:
+<div>
+  Hi <img src="{{this.model.profileImage}}" alt="{{this.model.name}}'s profile picture"> {{this.model.name}},
```

Comment:
This is where my desire for `@model` came up – it would be nicer if we could consistently introduce everything with `@args` and avoid `this.*` until we need to get into adding a class for components. Having model exposed as `this.model` but not `@model` breaks that flow.

cc @locks @rwjblue @wycats